themes/cms-forms/templates/SilverStripe/Forms/FormAction.ss

```diff
@@ -1,6 +1,6 @@
 <% if $UseButtonTag %>
 	<button $getAttributesHTML('class') class="btn<% if $extraClass %> $extraClass<% end_if %>">
-		<% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %>
+		<span><% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %></span>
```

Comment:
See comment on https://github.com/silverstripe/silverstripe-framework/pull/6907#discussion_r116392449

If we set html directly via $ButtonContent, then we shouldn't wrap that in a span.